.github/workflows/build-test-javascript.yaml

```diff
@@ -19,42 +18,79 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: true
+      - name: Set up tree-sitter
+        run: |
+          (cd libs/ocaml-tree-sitter-core && ./configure && ./scripts/install-tree-sitter-lib)
+      - name: Cache git checkout
```

Comment:
Do we need this? Does it improve significantly the build time?

Reply:
40+ seconds to do a full clone vs. 4s to restore repository from cache